src/views/project/publishPartition/SummaryView.vue

```diff
@@ -0,0 +1,177 @@
+<script setup>
+import axios from 'axios'
+import router from '@/router'
+import { onMounted, ref } from 'vue'
+import { useRoute } from 'vue-router'
+import LoadingIndicator from '@/components/project/LoadingIndicator.vue'
+import { useProjectsStore } from '@/stores/ProjectsStore'
+
+const route = useRoute()
+const projectId = route.params.id
+const partitionId = route.params.partitionId
+const isLoaded = ref(false)
+
+const projectStore = useProjectsStore()
+const project = ref(null)
+
+let partition = null
+let taxa = ref([])
+let characters = ref([])
+let onetimeMedia = ref([])
+let media = ref([])
+let labels = ref([])
+let documents = ref(0)
+let bibliographicReferences = ref(0)
+let views = ref([])
+let specimens = ref([])
+
+onMounted(async () => {
+  await projectStore.fetchProjects()
```

Comment:
This is unconditionally fetch all the projects - possibly again even if the project is already loaded. We check to see whether the store is loaded and if it's not we fetch the projects.

```
  if (!projectsStore.isLoaded) {
    projectsStore.fetchProjects()
  }
```